os/arch/arm/src/common/up_internal.h

```diff
@@ -122,12 +129,43 @@
 #undef CONFIG_RAMLOG_SYSLOG
 #endif
 
+
+/* ARM requires at least a 4-byte stack alignment.  For use with EABI and
+ * floating point, the stack must be aligned to 8-byte addresses.
+ */
+
+#ifndef CONFIG_STACK_ALIGNMENT
+#ifdef __ARM_EABI__
+#define CONFIG_STACK_ALIGNMENT 8
+#else
```

Comment:
This CONFIG_STACK_ALIGNMENT and STACK_ALIGNMENT also exist.

Reply:
Removed it as it is not required.